netstandard/ref/System.Reflection.cs

```diff
@@ -161,6 +162,12 @@ public sealed partial class AssemblyDescriptionAttribute : System.Attribute
         public AssemblyDescriptionAttribute(string description) { }
         public string Description { get { throw null; } }
     }
+    public static partial class AssemblyExtensions
```

Comment:
These are currently implemented in the System.Runtime.Loader contract in .NET Core. So we should decide if we want to separate these and make them supported everywhere or keep them in the Loader bucket and exclude them.

Reply:
I take that back this comes from System.Reflection.TypeExtensions https://github.com/dotnet/corefx/blob/a92474e2f5282fc2ac81c4f6d703b6d2f5248bac/src/System.Reflection.TypeExtensions/ref/System.Reflection.TypeExtensions.cs#L12 I confused it with the other type named AssemblyExtensions https://github.com/dotnet/corefx/blob/ec2a6190efa743ab600317f44d757433e44e859b/src/System.Runtime.Loader/ref/System.Runtime.Loader.cs#L11 in loader.

The question of whether or not to include System.Reflection.TypeExtensions is still interesting as it mostly duplicated what was already in Reflection so there is little value in including them.